server/controllers/finance/vouchers.js

```diff
@@ -193,12 +208,36 @@ function report(req, res, next) {
 }
 
 /**
+* GET /vouchers/reports
+*
+* @method report
+*/
+function report(req, res, next) {
+  'use strict';
+
+  let reportUrl = './server/controllers/finance/reports/voucher.report.handlebars';
+  let reportOptions = { pageSize : 'A4', orientation: 'landscape' };
+  let resolve = {
```

Comment:
Instead of adding a new API parameter to the `rm.build()` function, a better approach would use the `data` parameter in `rm.build()`.

Like this:
```js
.then(rows => {
  const data = { rows : rows, dateFrom: req.query.dateFrom, dateTo: req.query.dateTo };
  return rm.build(req, data, reportUrl, reportOptions);
})
.spread(/* ... etc ... */);
```

You would then update your handlebars files to use `data.rows` and `data.dateFrom`, etc.  This is much cleaner than adding an extra API method.